### PR TITLE
Switching collaborating with team and newcomers episodes

### PR DESCRIPTION
**Pull request by anenadic**
Switched collaborating with team and newcomers episodes. Extracted text on management and governance to a new episode. Fixes issue #128. There is still the bit about Milestones and Project Boards (#229) to be addressed after this PR is approved/merged.

I snuck in a new callout "Effective review comments are a learned skill" (to provide some guidance on doing PRs) at the end of the "Submitting and Reviewing Pull Requests on GitHub" section - this should have technically been a separate issue and a PR but I felt it might be easier to review here 😊 . I may have also opened a can of worms here as what should go in this callout is probably a separate discussion (and hence a separate PR would have been better 😢).

### CODE REVIEW

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:26]:
```suggestion
In this episode we expand on how to use GitHub effectively among a group of known
collaborators, currently your lesson development group, on an open source lesson development project, setting the stage for introducing tools and practices for
```

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:37]:
```suggestion
When you create an issue, you can add a range of details to it. An issue can be assigned to a specific team member - this can be a helpful way to know who is currently working to fix an issue or a way to assign responsibility. You can assign one or more labels to issues to classify or group them. Labels can also be given colours, allowing you to see at a glance from the Issues page the state of your project. The default labels available in GitHub include:
```

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:47]: Maybe we should switch this to be a list of the Carpentries labels and then add how to add them into the repo?

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:49]:
```suggestion
Check the [GitHub documentation on Issues](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) for the full reference.
```

File: episodes/collaborating-team.md
Comment (tobyhodges) [inline episodes/collaborating-team.md:228]:
```suggestion

The Turing Way has more advice on [Reviewing Contributions](https://the-turing-way.netlify.app/collaboration/maintain-review/maintain-review-review).

```

File: episodes/collaborating-team.md
Comment (tobyhodges) [inline episodes/collaborating-team.md:218]:
```suggestion
## Reviewing is a Skill
```

A shorter title, that has the advantage of following a theme from Instructor Training.

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:139]: This can wait as a separate PR but in #193 we discussed making this a call out instead of a demo and that could also be added to this PR if we wanted to

**Review by tobyhodges — APPROVED** [review]
This breakdown makes a lot of sense to me, and I think the order you have chosen for the episodes is good. I made a couple of suggestions related to the new callout, but happy for this to be merged with or without them.

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:227]:
```suggestion
- Good reviews have empathy - consider the contributor's feelings, identify positives and try not to be judgemental.
- Aim for improved/better rather than perfect.
- Make sure your review comments are specific and actionable.
- Make it clear in the comment what you want changed as part of a PR and, ideally, provide a concrete suggestion.
- Only request changes if the content is wrong or hard to understand (remember, different does not always mean better).
- If a conversation on a PR hasn’t been resolved by one or two back-and-forth exchanges, consider scheduling a meeting to discuss further.
```
Think we should put the empathy and better comments up front so they encounter them first.

Comment (sstevens2) [reply episodes/collaborating-team.md:227]: I really love the addition of this section, glad you included it in the PR

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:245]: You can mention people who aren't linked with the repo already but it won't list them so you will need to know their github username

File: episodes/collaborating-team.md
Comment (sstevens2) [inline episodes/collaborating-team.md:247]:
```suggestion
[Slack](https://slack.com/) is commonly used in the Carpentries community for quick, day-to-day message exchange among teams. You can create a channel for your lesson development project under [the Carpentries public Slack workspace](https://carpentries.slack.com/) or if you need more complex team communications you can create your own [Slack workspace for free](https://slack.com/intl/en-gb/). Note that the Carpentries Slack is an enterprise workspace so all messages and files will be retained and no messages will be lost (for free workspaces only the most recent 10,000 messages can be viewed and searched and file storage limit is 5 GB).
```
Think it is probably better practice for most groups to use the Carpentries workspace so moving that up front.

**Comment by tobyhodges** [comment]
(Assuming you approve of the change) if you can merge #259 into this branch before this one gets merged, that would be great.

File: episodes/collaborating-team.md
Comment (anenadic) [reply episodes/collaborating-team.md:245]: Good point - will include this @sstevens2.

Comment (anenadic) [inline episodes/collaborating-team.md:245]:
```suggestion
Typing `@` in GitHub will bring up a list of all accounts and teams linked to the repository that can be "mentioned". You can also mention people who aren't linked with the repository but they won't be listed in 
the `@` drop-down list so you will need to know their GitHub username in order to mention them.
```

File: episodes/collaborating-team.md
Comment (tobyhodges) [reply episodes/collaborating-team.md:47]: Or link out to that list? It is quite long...

Comment (tobyhodges) [reply episodes/collaborating-team.md:47]: https://docs.carpentries.org/topic_folders/maintainers/github_labels.html

File: episodes/collaborating-team.md
Comment (anenadic) [reply episodes/collaborating-team.md:139]: This is now a separate PR  https://github.com/carpentries/lesson-development-training/pull/259 that will get merged on top of this one.

File: episodes/collaborating-team.md
Comment (sstevens2) [reply episodes/collaborating-team.md:47]: Moving to own issue

Comment (sstevens2) [reply episodes/collaborating-team.md:47]: Now issue #260

File: episodes/collaborating-team.md
Comment (anenadic) [inline episodes/collaborating-team.md:236]: If you need more help or practice with GitHub features, you can attend a GitHub skill-up session for Carpentries Maintainers and lesson developers which uses the [following curriculum](https://carpentries.github.io/github-skill-up-maintainers/). These sessions will be periodically announced on [The Carpentries calendar of events](https://carpentries.org/community/#community-events), [The Carpentries Slack]((https://carpentries.slack.com/)) and mailing lists.(https://carpentries.github.io/github-skill-up-maintainers/).

**Pull request merged**
